src/library/crate/cratestorage.cpp

```diff
@@ -440,6 +453,23 @@ CrateTrackSelectResult CrateStorage::selectTrackCratesSorted(TrackId trackId) co
     }
 }
 
+CrateTrackSelectResult CrateStorage::selectTracksSortedByCrateNameLike(const QString& crateNameLike) const {
+    FwdSqlQuery query(m_database, QString(
+            "SELECT DISTINCT %1,%2 FROM %3 JOIN %4 ON %5 = %6 WHERE %7 LIKE %8 ORDER BY %1").arg(
+                    CRATETRACKSTABLE_TRACKID,
+                    CRATETRACKSTABLE_CRATEID,
+                    CRATE_TRACKS_TABLE,
+                    CRATE_TABLE,
+                    CRATETABLE_ID,
+                    CRATETRACKSTABLE_CRATEID,
+                    CRATETABLE_NAME,
+                    crateNameLike));
```

Comment:
Please bind crateNameLike as a parameter by-value to the query instead of formatting it into the string. Then you don't need to escape it manually
"...%7 LIKE :crateNameLike..."
query.bindValue(":crateNameLike", crateNameLike);